src/js/shared.js

```diff
@@ -623,11 +625,14 @@ function cardButtons (t) {
                                 memberRanges.forEach((range) => {
                                     const start = new Date(range.item.start * 1000);
                                     const end = new Date(range.item.end * 1000);
+									const rangeOnTheSameDay = start.toDateString() === end.toDateString();
+									const rangeLengthInSeconds = range.item.end - range.item.start;
```

Comment:
@jonez1 It appears that you've used tabs. Can you change this to spaces? Just so it's in line with the rest of the code base.